pkg/console/subresource/deployment/deployment.go

```diff
@@ -154,6 +171,27 @@ func Stub() *appsv1.Deployment {
 	return dep
 }
 
+func LogDeploymentAnnotationChanges(client appsclientv1.DeploymentsGetter, updated *appsv1.Deployment) {
+	existing, err := client.Deployments(updated.Namespace).Get(updated.Name, metav1.GetOptions{})
+	if err != nil {
+		return
```

Comment:
Shouldn't we log the error?

Reply:
I can. The caller has a good bit of logging around it (`SyncDeployment`) but no point in coupling.